adapters/polychain-monsters/index.js

```diff
@@ -0,0 +1,115 @@
+require("dotenv").config();
+
+const moment = require("moment");
+const BigNumber = require("bignumber.js");
+const Ethereum = require("../../sdk/EVMC");
+const axios = require("axios");
+const URL = "http://nft-sales-service.dappradar.com/open-source";
+const KEY = process.env.DAPPRADAR_API_KEY;
+const path = require("path");
+const { AVAILABLE_PROTOCOLS } = require("../../sdk/util");
+
+class PolychainMonsters {
+    constructor() {
+        this.name = "polychain-monsters";
+        this.token = "0x1796ae0b0fa4862485106a0de9b654efe301d0b2";
+        this.protocol = AVAILABLE_PROTOCOLS.BSC;
+        this.block = 8482954;
+        this.contract = "0x85F0e02cb992aa1F9F47112F815F519EF1A59E2D";
+        this.events = ["Transfer"];
+        this.pathToAbi = path.join(__dirname, "./abi.json");
+        this.range = 500;
+        this.chunkSize = 6;
+        this.sdk = new Ethereum(this);
+    }
+
+    run = async () => {
+        const s = await this.getSymbol();
+        this.sdk = this.loadSdk();
+        this.symbol = s;
+        await this.sdk.run();
+    };
+
+    loadSdk = () => {
+        return new Ethereum(this);
```

Comment:
For binance adapters you need to use the binance SDK. I have just commited it so it would be easier for you.

Reply:
Of course, thanks a lot!